src/components/Home.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Typewriter from './Typewriter';
+
+function Home() {
+  return (
+    <div className="home">
+      <h1
+        className="home__title
+  "
+      >
+        Welcome
+      </h1>
+      {/* <p className="home__subtitle">That is a small test blog made by Alexander Bakay</p> */}
+      <Typewriter>Test blog by Alexander Bakay</Typewriter>
+    </div>
+  );
+}
+
+export default Home;
```

Comment:
also you can export your component or const or something else not default,  like this `export const Example = () => {return (<></>)}`
and then import { Example } from '../Example'

Reply:
understand, but this is a result of short "rfce" command

Reply:
like you want)